schemas/io-package.json

```diff
@@ -0,0 +1,457 @@
+{
+    "$schema": "http://json-schema.org/schema#",
+    "type": "object",
+    "required": ["common"],
+    "additionalProperties": false,
+    "properties": {
+        "$schema": {
+            "description": "JSON schema for validation",
+            "type": "string"
+        },
+        "common": {
+            "required": ["name", "version", "platform", "titleLang", "news", "desc", "mode"],
+            "additionalProperties": false,
+            "description": "Contains all common instance attributes",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "description": "Name of the adapter without ioBroker",
+                    "type": "string",
+                    "pattern": "(?i)^(?!iobroker)"
+                },
+                "version": {
+                    "description": "Current adapter version",
+                    "type": "string",
+                    "pattern": "^\\d+\\.\\d+\\.\\d+.*"
```

Comment:
==> https://github.com/semver/semver/blob/master/semver.md#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string

Reply:
lol nice regex, will add it